### PR TITLE
Feat/implement endpoint get doctors by hospital

### LINKED ISSUE #9: Add get doctors by hospital endpoint

### PR DESCRIPTION
**Pull request by aledeltoro**
## Issue
Fixes #9 
## Dependencies
- https://github.com/HospiCloud-Team/hospicloud-backend-api/pull/17
## Tasks
- [x] Add endpoint to return doctors by hospital ID
- [x] Add test for new endpoint 
- [x] Add logic to handle `description` field in `Hospital` schema

### CODE REVIEW

**Review by cesarlopezt — APPROVED** [review]
Looks good to me 👍

File: common/schemas/user.py [hunk 1]
```diff
@@ -33,12 +33,40 @@ class UserBase(BaseModel):
...
     id: int
     patient: Optional[Patient] = None
     admin: Optional[Admin] = None
+    doctor: Optional[Doctor] = None
+
+    class Config:
+        orm_mode = True
+
+
+class DoctorOutput(DoctorBase):
```
Comment (frankroberto20) [inline common/schemas/user.py:59]: En vez de DoctorOutput por que no DoctorOut?

Comment (aledeltoro) [reply common/schemas/user.py]: Done

Comment (aledeltoro) [reply common/schemas/user.py]: Done

**Pull request merged**